<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KScope-NG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCopy_As_HTML"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect"/>
    <addaction name="actionBlock_Selection"/>
    <addaction name="separator"/>
    <addaction name="actionOverwrite_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Previous"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_Line"/>
    <addaction name="actionSearch_Incrementally"/>
    <addaction name="actionSearch_Incrementally_Backwards"/>
    <addaction name="actionGo_to_Tag"/>
    <addaction name="actionComplete_Symbol"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionReload"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExport_as_HTML"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToggleFiles"/>
    <addaction name="actionToggleCscope"/>
    <addaction name="actionToggleSymbols"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCscope">
    <property name="title">
     <string>Cscope</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuProject"/>
   <addaction name="menuCscope"/>
   <addaction name="menuGo"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionPrint"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="actionEnlarge_Font"/>
   <addaction name="actionShrink_Font"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockCscope">
   <property name="windowTitle">
    <string>CScope</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockCscopeContents">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableWidget" name="tableResults"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockFiles">
   <property name="windowTitle">
    <string>Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockFilesContents">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QTreeWidget" name="treeFiles">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockSymbols">
   <property name="windowTitle">
    <string>Symbols</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockSymbolsContents">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QTreeWidget" name="treeSymbols">
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>70</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>struct</string>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int</string>
        </property>
        <property name="text">
         <string>Variable_Name</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="projectToolBar">
   <property name="windowTitle">
    <string>Project Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenProject"/>
   <addaction name="actionRebuildProject"/>
  </widget>
  <widget class="QToolBar" name="navigationToolBar">
   <property name="windowTitle">
    <string>Navigation Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPreviousPosition"/>
   <addaction name="actionNextPosition"/>
   <addaction name="actionPositionHistory"/>
   <addaction name="actionGlobalBookmarks"/>
  </widget>
  <widget class="QToolBar" name="queryToolBar">
   <property name="windowTitle">
    <string>Query Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewQuery"/>
   <addaction name="actionRefreshQueries"/>
   <addaction name="actionLock"/>
   <addaction name="actionClose_2"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
  </widget>
  <widget class="QToolBar" name="dockToolBar">
   <property name="windowTitle">
    <string>Workspace Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleSymbols"/>
   <addaction name="actionToggleCscope"/>
   <addaction name="actionToggleFiles"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/close.png</normaloff>:/icons/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/saveas.png</normaloff>:/icons/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/redo.png</normaloff>:/icons/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/cut.png</normaloff>:/icons/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/copy.png</normaloff>:/icons/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/paste.png</normaloff>:/icons/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCopy_As_HTML">
   <property name="text">
    <string>Copy As HTML</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="text">
    <string>Deselect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionBlock_Selection">
   <property name="text">
    <string>Block Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/saveall.png</normaloff>:/icons/icons/saveall.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/reload.png</normaloff>:/icons/icons/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/print.png</normaloff>:/icons/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport_as_HTML">
   <property name="text">
    <string>Export as HTML</string>
   </property>
  </action>
  <action name="actionOverwrite_Mode">
   <property name="text">
    <string>Overwrite Mode</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/find.png</normaloff>:/icons/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/next.png</normaloff>:/icons/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFind_Previous">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/previous.png</normaloff>:/icons/icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEnlarge_Font">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/increasefont.png</normaloff>:/icons/icons/increasefont.png</iconset>
   </property>
   <property name="text">
    <string>Enlarge Font</string>
   </property>
  </action>
  <action name="actionShrink_Font">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/decreasefont.png</normaloff>:/icons/icons/decreasefont.png</iconset>
   </property>
   <property name="text">
    <string>Shrink Font</string>
   </property>
  </action>
  <action name="actionGo_to_Line">
   <property name="text">
    <string>Go to Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSearch_Incrementally">
   <property name="text">
    <string>Search Incrementally</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionSearch_Incrementally_Backwards">
   <property name="text">
    <string>Search Incrementally Backwards</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+F</string>
   </property>
  </action>
  <action name="actionGo_to_Tag">
   <property name="text">
    <string>Go to Tag</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionComplete_Symbol">
   <property name="text">
    <string>Complete Symbol</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionToggleSymbols">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/taglist.png</normaloff>:/icons/icons/taglist.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Tag List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+'</string>
   </property>
  </action>
  <action name="actionToggleCscope">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/querywindow.png</normaloff>:/icons/icons/querywindow.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Query Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionToggleFiles">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/filelist.png</normaloff>:/icons/icons/filelist.png</iconset>
   </property>
   <property name="text">
    <string>Toggle File List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/project_open.png</normaloff>:/icons/icons/project_open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionRebuildProject">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/rebuild.png</normaloff>:/icons/icons/rebuild.png</iconset>
   </property>
   <property name="text">
    <string>Rebuild Project</string>
   </property>
   <property name="toolTip">
    <string>Rebuild Project</string>
   </property>
  </action>
  <action name="actionPreviousPosition">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/back.png</normaloff>:/icons/icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Previous Position</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionNextPosition">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/forward.png</normaloff>:/icons/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Next Position</string>
   </property>
   <property name="toolTip">
    <string>Next Position</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionGlobalBookmarks">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/bookmark.png</normaloff>:/icons/icons/bookmark.png</iconset>
   </property>
   <property name="text">
    <string>Global Bookmarks</string>
   </property>
  </action>
  <action name="actionPositionHistory">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/history.png</normaloff>:/icons/icons/history.png</iconset>
   </property>
   <property name="text">
    <string>Position History</string>
   </property>
  </action>
  <action name="actionNewQuery">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New Query</string>
   </property>
  </action>
  <action name="actionRefreshQueries">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/reload.png</normaloff>:/icons/icons/reload.png</iconset>
   </property>
   <property name="text">
    <string>RefreshQueries</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/lock.png</normaloff>:/icons/icons/lock.png</iconset>
   </property>
   <property name="text">
    <string>Lock Query</string>
   </property>
  </action>
  <action name="actionClose_2">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/close.png</normaloff>:/icons/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close Query</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/up.png</normaloff>:/icons/icons/up.png</iconset>
   </property>
   <property name="text">
    <string>Previous Result</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../res/resources.qrc">
     <normaloff>:/icons/icons/down.png</normaloff>:/icons/icons/down.png</iconset>
   </property>
   <property name="text">
    <string>Next Result</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
